packages/react-native-client/src/hooks/usePeers.ts

```diff
@@ -8,42 +8,44 @@ import {
 import RNFishjamClientModule from '../RNFishjamClientModule';
 import { ReceivableEvents, eventEmitter } from '../common/eventEmitter';
 
+export type Peer<
+  MetadataType extends Metadata,
+  VideoTrackMetadataType extends Metadata,
+  AudioTrackMetadataType extends Metadata,
+> = Endpoint<MetadataType, VideoTrackMetadataType, AudioTrackMetadataType>;
+
 /**
- * This hook provides live updates of room endpoints.
- * @returns An array of room endpoints.
+ * This hook provides live updates of room peers.
+ * @returns An array of room peers.
  */
-export function useEndpoints<
-  EndpointMetadataType extends Metadata,
+export function usePeers<
```

Comment:
Notice that they're going to be components like here: https://github.com/fishjam-cloud/web-client-sdk/blob/main/packages/ts-client/src/FishjamClient.ts#L537 (they're going to be soon in mobile?)
I still don't quite understand the difference between components and peers and how user handles it, but probably there is going to be a lot of code reuse.
In particular if user handles componenets and peers in the same way then maybe there is no point in making a distinction between peers and components and we should just leave the endpoints?